test/libsolidity/syntaxTests/literalSuffixes/parameters/type_mismatch.sol

```diff
@@ -66,41 +66,41 @@ contract C {
     }
 }
 // ----
-// TypeError 8838: (635-647): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (657-672): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (721-735): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (745-758): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (808-823): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (833-848): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (858-876): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (925-942): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (952-968): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (979-994): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (1004-1019): The type of the literal cannot be converted to the parameter of the suffix function.
-// TypeError 8838: (1068-1086): The type of the literal cannot be converted to the parameter of the suffix function.
+// TypeError 8838: (635-647): The number cannot be converted to type bool accepted by the suffix function.
+// TypeError 8838: (657-672): The number cannot be converted to type address accepted by the suffix function.
+// TypeError 8838: (721-735): The number cannot be converted to type string accepted by the suffix function.
+// TypeError 8838: (745-758): The number cannot be converted to type bytes accepted by the suffix function.
+// TypeError 8838: (808-823): The number is out of range of type int8 accepted by the suffix function.
+// TypeError 8838: (833-848): The number cannot be converted to type bool accepted by the suffix function.
+// TypeError 8838: (858-876): The number cannot be converted to type address accepted by the suffix function.
+// TypeError 8838: (925-942): The number cannot be converted to type string accepted by the suffix function.
+// TypeError 8838: (952-968): The number cannot be converted to type bytes accepted by the suffix function.
+// TypeError 8838: (979-994): The literal cannot be converted to the type accepted by the suffix function.
+// TypeError 8838: (1004-1019): The literal cannot be converted to the type accepted by the suffix function.
+// TypeError 8838: (1068-1086): The literal cannot be converted to the type accepted by the suffix function.
```

Comment:
Here we're mentioning the parameter type only when the literal is an integer. Would be best to mention it for all types.

The reorganization for one of the `if` is already mentioned before should fix it, but just pointing it out here so that I remember to recheck the output.